pkg/process/checks/process_linux.go

```diff
@@ -0,0 +1,61 @@
+// Unless explicitly stated otherwise all files in this repository are licensed
+// under the Apache License Version 2.0.
+// This product includes software developed at Datadog (https://www.datadoghq.com/).
+// Copyright 2016-present Datadog, Inc.
+
+//go:build linux
+
+package checks
+
+import (
+	workloadmetacomp "github.com/DataDog/datadog-agent/comp/core/workloadmeta/def"
+	"github.com/DataDog/datadog-agent/pkg/process/procutil"
+)
+
+func mapWLMProcToProc(wlmProc *workloadmetacomp.Process, stats *procutil.Stats) *procutil.Process {
```

Comment:
helpers go at the bottom per Will's comment:
https://github.com/DataDog/datadog-agent/pull/38817#discussion_r2222991656